tracer/src/Datadog.Trace/Generated/net461/Datadog.Trace.SourceGenerators/Datadog.Trace.SourceGenerators.TelemetryMetric.TelemetryMetricGenerator/DistributionExtensions.g.cs

```diff
@@ -7,7 +7,7 @@ internal static partial class DistributionExtensions
     /// <summary>
     /// The number of separate metrics in the <see cref="Datadog.Trace.Telemetry.Metrics.Distribution" /> metric.
     /// </summary>
-    public const int Length = 1;
+    public const int Length = 14;
```

Comment:
I assume you know already as you work with Andrew on this but - just in case - we don't set distribs yet. Still make sense to have it added in that PR obviously